### PR TITLE
[FEATURE]Add action to process stale pull requests.

### PR DESCRIPTION
**Pull request by FMX**
# [FEATURE]Add action to process stale pull requests.

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by FMX** [comment]
ping @waitinfuture @pan3793 @AngersZhuuuu . I want to add an action to close stale pull requests.

File: .github/workflows/check_stale_prs.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-pr-stale: 20
```
Comment (waitinfuture) [inline .github/workflows/check_stale_prs.yml:23]: I'm not sure whether we should stale or close a PR first, is there an example?

**Comment by FMX** [comment]
@waitinfuture Conversation resolved.

**Comment by FMX** [comment]
Hold for a second.

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**